src/components/Profile/Profile.js

```diff
@@ -8,7 +8,7 @@ import style from './Profile.module.css';
 
 const Profile = () => {
     return (
-        <div className='content'>
+        <div>
```

Comment:
Now you have two nested divs - is that intentional?
Consider either omitting one or replace it with Fragment.

Reply:
/fxd. This was a necessary preparation, now eliminated